configurations/CAndy/dev/allowed-schema.csv

```diff
@@ -1,4 +1,4 @@
 author_did,schema_name,version,details
 9defyjkM6MX5zh2D5Mwo1U,contractor-credential,*,https://github.com/bcgov/digital-trust-toolkit/blob/contractor-credential/docs/governance/employment/contractor-credential/governance.md#261-schema-definition
-RCnz8GcyZ2iH7VFr5zGb9N,Member Card,*,
+RCnz8GcyZ2iH7VFr5zGb9N,*,*,
```

Comment:
Wonder if we should put details in this one? Since no naming doesn't show what it is without looking up the DID

![image](https://github.com/user-attachments/assets/ae2dc60d-1606-4986-a47b-581e6e3355f2)

Reply:
I agree with @loneil

Reply:
Good idea, done.